cmd/restic/cmd_backup.go

```diff
@@ -645,6 +645,7 @@ func runBackup(ctx context.Context, opts BackupOptions, gopts GlobalOptions, ter
 		Time:           timeStamp,
 		Hostname:       opts.Host,
 		ParentSnapshot: parentSnapshot,
+		ProgramVersion: version,
```

Comment:
Shouldn't this be

```suggestion
		ProgramVersion: "restic " + version,
```

so that snapshots from restic and rustic can be easily distinguished? Just a question, restic is of course the reference implementation.

Reply:
Good idea. I've total missed that `version` doesn't include the program name.